src/bun.js/child_process.exports.js

```diff
@@ -1,3 +1,1005 @@
+const EventEmitter = import.meta.require("node:events");
+const { Readable } = import.meta.require("node:stream");
+const {
+  constants: { signals },
+} = import.meta.require("node:os");
+const assert = import.meta.require("node:assert");
```

Comment:
`assert` is big for just one function call

Reply:
Totally, this is a silly use case, ✂️ing this as well